program/process/OpenHW Group IP Core - RTL Freeze Checklist and Release Process.md

```diff
@@ -0,0 +1,130 @@
+# OpenHW Group IP Core - RTL Freeze Checklist and Release Process
+
+
+This document describes the release process used by OpenHW Group for IP cores projects.
+
+In this process, OpenHW validates that a set of RTL Freeze checklist tasks have been completed prior to release.
+Checklist completion enables the creation and publishing of one or more "tags" on the project repositories. A consumer of the release accesses the tagged code associated with the release via the tags. OpenHW publishes the release name and description along with the release tags.
+
+
+The following aspects of the release process are described in this document: 
+* Naming the RTL Freeze release

```

Comment:
I would suggest putting links to the sections. 
```suggestion
* [Naming the RTL Freeze release](#-naming-the-release)
```
https://gist.github.com/rachelhyman/b1f109155c9dafffe618